ngi_reports/common/ign_sample_report.py

```diff
@@ -48,8 +49,9 @@ def __init__(self, config, LOG, working_dir, **kwargs):
         self.parse_picard_metrics()
 
         # Plot graphs
-        self.LOG.info('Plotting graphs')
-        self.make_plots()
+        if create_plots:
```

Comment:
It took quite long to create the plots compared to just creating the csv, and I suspect that the time will be longer the larger the project. So I thought it was nice to add the possibility of skipping it to speed things up.